azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageVpnGatewayPoint2SiteConnection.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.network.samples;
+
+import com.microsoft.azure.management.Azure;
+import com.microsoft.azure.management.network.Network;
+import com.microsoft.azure.management.network.VirtualNetworkGateway;
+import com.microsoft.azure.management.network.VirtualNetworkGatewaySkuName;
+import com.microsoft.azure.management.resources.fluentcore.arm.Region;
+import com.microsoft.azure.management.resources.fluentcore.utils.SdkContext;
+import com.microsoft.azure.management.samples.Utils;
+import com.microsoft.rest.LogLevel;
+
+import java.io.File;
+
+/**
+ * Azure Network sample for managing virtual network gateway.
+ *  - Create a virtual network with subnets
+ *  - Create virtual network gateway
+ *  - Update virtual network gateway with Point-to-Site connection configuration
+ *  - Generate and download VPN client configuration package. Now it can be used to create VPN connection to Azure.
+ *  - Revoke a client certificate
+ *
+ *  Please note: in order to run this sample, you need to have:
+ *   - pre-generated root certificate and public key exported to $CERT_PATH file
+ *      For more details please see https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-certificates-point-to-site for PowerShell instructions
+ *      and https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-certificates-point-to-site-makecert for Makecert instructions.
+ *   - client certificate generated for this root certificate installed on your machine.
+ *      Please see: https://docs.microsoft.com/en-us/azure/vpn-gateway/point-to-site-how-to-vpn-client-install-azure-cert
+ *   - thumbprint for client certificate saved to $CLIENT_CERT_THUMBPRINT
```

Comment:
to be consistent with wording for `$CERT_PATH`.. "thumbprint for client certificate exported to $CLIENT_CERT_THUMBPRINT"